disco_aws_automation/disco_logging.py

```diff
@@ -3,12 +3,20 @@
 import sys
 
 
-def configure_logging(debug):
+def configure_logging(debug, silent=False):
     '''Sets the default logger and the boto logger to appropriate levels of chattiness.'''
     logger = logging.getLogger('')
     boto_logger = logging.getLogger('boto')
     botocore_logger = logging.getLogger('botocore')
-    if debug:
+
+    if silent and debug:
+        raise Exception('Debug and silent logging options are mutually exclusive')
+
+    if silent:
+        logger.disabled = True
```

Comment:
Default `logger.disabled` is of type `int` and value `0`. Wouldn't we set `logger.disabled` to `1` instead of `True`?